person.rb

```diff
@@ -25,6 +26,11 @@ def can_use_services?
 
   private
 
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] For add_rental you should call the parameters `book, date`, and then the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association 🙃.
It can be similar to:

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```